domain/src/test/java/org/oppia/android/domain/topic/TopicListControllerTest.kt

```diff
@@ -78,13 +79,14 @@ class TopicListControllerTest {
   lateinit var mockTopicListObserver: Observer<AsyncResult<TopicList>>
 
   @Mock
-  lateinit var mockOngoingStoryListObserver: Observer<AsyncResult<OngoingStoryList>>
+  lateinit var mockPromotedActivityListObserver: Observer<AsyncResult<PromotedActivityList>>
```

Comment:
Overall, please check/add:
- Tests for older/recently played (if these aren't already covered in existing tests)
- Mix of upcoming/recently played
- The scenario where a topic is completed but another one of its stories has been started (we should also be verifying that the new bit in the proto is being set correctly in this situation & specifically not set for other test cases)

Reply:
- Tests for older/recently played are covered. 
- Mix of **upcoming/recently played** we are not showing this combination right? Or I have missed it?

Reply:
Ah sorry. I meant a combination of recently played + suggested, not upcoming. We should make sure upcoming specifically is tested, too.

Reply:
Yes these testcases are covered.

Reply:
Hmm I don't see a test for the case where a topic has one story completed & another one of its stories started. Is that case being covered?

Reply:
Done.